### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by carlioth**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CALLBACK_URL=https://ap.core.qlik.com
+BACKEND_ADRESS=apps.core.qlik.com/secure/doc/fc649d13-84ea-4174-8c96-67208f069587
```
Comment (peol) [inline .env]: Typo `ADDRESS`. I'm guessing this is the full websocket URL, maybe include `wss://`?

File: src/index.template.html [hunk 2]
```diff
@@ -4,10 +4,24 @@
...
   <meta charset="UTF-8">
   <title>Assisted Prescription</title>
+  <script>(function(w,d,s,l,i){
+    w[l]=w[l]||[];
+    w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});
+    var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';
+    j.async=true;
+    j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;
+    f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer', 'GTM-P7VJSX');
```
Comment (peol) [inline src/index.template.html]: Maybe expose ID as a process.env variable?

File: src/main.jsx [hunk 3]
```diff
@@ -23,100 +20,68 @@ picasso.use(picassoQ);
...
     };
 
-    // auth.idp.then((authMode) => {
-    //   this.setState({ authMode });
-    // });
-
-    // auth.isAuthenticated.then((authenticated) => {
-    //   this.setState({ isAuthenticated: authenticated });
-    // });
+    auth.isAuthenticated.then((authenticated) => {
```
Comment (peol) [inline src/main.jsx]: If you rename this to `isAuthenticated` you can do `isAuthenticated => this.setState({ isAuthenticated })`

File: .env (hunk 1 above)
Comment (carlioth) [reply .env]: typo fixed. The adress is transformed `window.location.protocol.replace('http', 'ws'`

File: src/index.template.html (hunk 2 above)
Comment (carlioth) [reply src/index.template.html]: Moved it to webpack.config, and then template it

**Comment by carlioth** [comment]
These changes make this on par with the swarm example. An issue to make this code more easy to run locally has been created: https://github.com/qlik-oss/core-assisted-prescription-ui/issues/321

**Pull request merged**